dictionary.json

```diff
@@ -18137,19 +18846,13 @@
   },
   {
     "toaq": "za",
-    "type": "predicate",
-    "english": "▯ is about to be the case.",
-    "gloss": "[inchoative]",
-    "gloss_abbreviation": "INCH",
+    "type": "aspect",
+    "english": "prospective (have yet to)",
```

Comment:
IMO "going to" is a better match than "have yet to", since the latter can be mistaken for "bu haı"